featauthentication/src/main/java/com/praxis/feat/authentication/vm/AuthVM.kt

```diff
@@ -42,17 +43,22 @@ class AuthVM @Inject constructor(
       .launchIn(viewModelScope)
   }
 
-  fun loginNow(onLoginNavigate: () -> Unit = {}) {
+  fun loginNow() {
+    formVisibility.value = false
+
     try {
       val isValid = credentials.value.validate()
       if (isValid) {
         snackBarState.value = ""
-        onLoginNavigate()
-        //navigator.navigate(Screen.Jokes.route)
+        viewModelScope.launch {
+          delay(1500)
+          navigator.navigateFragment(R.id.action_authFragment_to_viewPagerFragment)
```

Comment:
Moving this fragment-based navigation to Navigator module looks more meaningful now. 👍 💯